.github/workflows/ci.yml

```
@@ -51,137 +37,161 @@ jobs:
           activate-environment: srbench
           use-mamba: true
```

```suggestion
          use-mamba: true
          python-version: 3.11
```

Looks like the Python version was not set here, so recently updated to 3.12 which will break a bunch of packages that haven't yet released their 3.12 versions (PyTorch usually takes a while to do so, for example).